File: lib-widget/src/main/java/com/lucers/widget/WidgetActivity.java
package com.lucers.widget;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

/**
 * WidgetActivity
 *
 * @author lucerstu
 */
public class WidgetActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_widget);
        initView();
        initData();
    }

    private void initView() {
    }

    private void initData() {

    }
}

File: module-widget/src/main/java/com/lucers/widget/ui/WidgetActivity.kt
package com.lucers.widget.ui

import android.os.Bundle
import com.alibaba.android.arouter.facade.annotation.Route
import com.lucers.common.base.BaseActivity
import com.lucers.common.constants.AppRouteConstants
import com.lucers.widget.R

/**
 * WidgetActivity
 */
@Route(path = AppRouteConstants.widgetRoute)
class WidgetActivity : BaseActivity(R.layout.activity_widget) {

    override fun initView(savedInstanceState: Bundle?) {
        super.initView(savedInstanceState)
    }

    override fun initData(savedInstanceState: Bundle?) {
        super.initData(savedInstanceState)
    }
}
